openassessment/templates/openassessmentblock/grade/oa_assessment_feedback.html

```diff
@@ -1,41 +1,43 @@
 {% load i18n %}
 {% spaceless %}
     {% if assessment %}
+      {% if assessment.individual_assessments or assessment.feedback %}
         <li class="answer feedback">
-            {% if assessment.individual_assessments %}
-                {% for individual_assessment in assessment.individual_assessments %}
-                    {% if individual_assessment.feedback %}
-                        <div class="answer_title">
-                            <span class="answer__source">
-                                {%  if individual_assessment.option %}
-                                    {% blocktrans with title=individual_assessment.title grade=individual_assessment.option.label start_tag='<span class="answer__source__value">'|safe end_tag="</span>"|safe %}
-                                        {{ start_tag }}{{ title }}{{ end_tag }} - {{ grade }}
-                                    {% endblocktrans %}
-                                {% else %}
-                                    <span class="answer__source__value">
-                                        {{ individual_assessment.title }}
-                                    </span>
-                                {% endif %}
-                            </span>
-                        </div>
-
-                        <div class="feedback__value">
-                            <p class="feedback__value__raw">{{ individual_assessment.feedback }}</p>
-                        </div>
-                    {% endif %}
-                {% endfor %}
-            {% else %}
+          {% if assessment.individual_assessments %}
+            {% for individual_assessment in assessment.individual_assessments %}
+              {% if grade != '' %}
```

Comment:
When is the grade empty? If it is empty, does it makes sense to show the feedback (as there will be no title)?